Source/WebKit/Platform/ios/PlaybackSessionInterfaceLMK.mm

```diff
@@ -182,12 +182,18 @@ - (void)linearMediaPlayer:(WKSLinearMediaPlayer *)player setLegibleTrack:(WKSLin
 
 - (void)linearMediaPlayerToggleInlineMode:(WKSLinearMediaPlayer *)player
 {
-    auto model = _model.get();
-    if (!model)
-        return;
+    [self _exitFullscreen];
```

Comment:
It seems odd to call `_exitFullscreen` from `ToggleInlineMode`

Reply:
True. Since we set `allowFullScreenFromInline` on the player, this is how LinearMediaKit tells us that the inline/fullscreen button was pressed by the user. In practice though we never show the LMK controls in inline mode, so when this method is called it always means we are exiting fullscreen.

This does seem fragile though; I'll add an additional check here to make sure the `presentationMode` is really `fullscreen` before calling `-_exitFullscreen`.